editor/sidebar/post-visibility/index.js

```diff
@@ -100,9 +112,13 @@ class PostVisibility extends Component {
 						<div className="editor-post-visibility__dialog-legend">
 							{ __( 'Post Visibility' ) }
 						</div>
-						{ visibilityOptions.map( ( { value, label, info, changeHandler, checked } ) => (
+						{ visibilityOptions.map( ( { value, label, info, checked } ) => (
 							<label key={ value } className="editor-post-visibility__dialog-label">
-								<input type="radio" value={ value } onClick={ changeHandler } checked={ checked } />
+								<input
+									type="radio"
+									value={ value }
+									onChange={ this.setVisibility }
```

Comment:
Well, `onClick` is not the correct way to handle an input change. Not sure how to address the issue without knowing what it was, but I have a feeling there might be a way to resolve it while still keeping `onChange`. Let's keep an eye out for regressions.